BridgeAppSDK/SBATrackedDataObjectCollection.swift

```diff
@@ -232,4 +232,11 @@ extension SBATrackedDataObjectCollection: SBABridgeTask, SBAStepTransformer, SBA
         return selectedItems.find({ $0.tracking }) != nil
     }
 
+    open func commitChanges(user: SBAUserWrapper, task: SBANavigableOrderedTask, taskResult: ORKTaskResult) {
+        
```

Comment:
Is this meant to be overridden in subclasses? Implementing a protocol method as a stub? Something else? A quick comment would be helpful here